tests/cpp/test_multidevice_communications.cpp

```diff
@@ -13,6 +13,9 @@
 #include <multidevice/communicator.h>
 #include <tests/cpp/multidevice.h>
 
+#include <ops/arith.h>
+#include <ops/utils.h>
```

Comment:
just to be sure: is `ops/utils.h` used indeed ?

Reply:
yes--without ops/utils.h, ops::newValLike is a compiler error